content/en/docs/guides/server-graceful-stop.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Gracefull Shutdown
+description: >-
+  Explains how to gracefully stop/shutdown a gRPC server.
+---
+
+### Overview
+
+gRPC servers often need to shut down gracefully, ensuring that in-flight RPCs
+are completed within a reasonable timeframe and new RPCs are no longer
+accepted. The "Graceful shutdown function" facilitates this process, allowing
+the server to transition smoothly without abruptly terminating active
+connections.
+
+When the "Graceful shutdown function" is called, the server immediately stops
+accepting new RPCs. In-flight RPCs are allowed to continue until they complete
+or a specified deadline is reached. Once all active RPCs finish or the deadline
+expires, the server shuts down completely.
+
+While a graceful shutdown is optional, it's highly recommended. You can
+shutdown without a graceful shutdown first but that would not be recommended.
```

Comment:
"You can shut down without a graceful shutdown" (note verb = space, noun = no space).

Also let's avoid recommended twice in a row here.  Maybe:

```
Because graceful shutdown helps prevent clients from encountering RPC failures, it should be used if
possible.  However, gRPC also provides a forceful shutdown mechanism which will immediately cause the
server to stop serving and close all connections, which results in the failure of any in-flight RPCs.
```

Reply:
Updated